Makefile

```diff
@@ -193,7 +193,7 @@ docclean:
 run-ci:
 	$(PYTHON) ./configure $(CONFIG_FLAGS)
 	$(MAKE)
-	$(MAKE) test-ci
+	$(MAKE) test-ci test-addons
```

Comment:
Based on discussion in #2134 when the `run-ci` target was added, this seems like the right place to add it as long as the intention is to run these tests as part of every build done on CI.